.github/workflows/operator-integration-tests-v2.yml

```diff
@@ -202,7 +202,92 @@ jobs:
+          ref: ${{ needs.now-prep-testbed.outputs.tag }}
+      - name: Setup Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Prepare environment
+        run: |
+          sudo apt update && sudo apt install ffmpeg
+          python -m pip install --upgrade pip
+          python -m pip install wheel
+          export JCLOUD_LOGLEVEL=DEBUG
+          pip install --no-cache-dir ".[full,test]"
+      - name: Add JCloud auth token
+        run: |
+          mkdir -p ~/.jina
+          touch ~/.jina/config.json
+          echo "{\"auth_token\": \"${_WOLF_NPRD_GENERAL_WOLF_TOKEN}\"}" > ~/.jina/config.json
+          cat ~/.jina/config.json
+      - name: tmate session
+        uses: mxschmitt/action-tmate@v3
+      - name: Run tests
+        run: |
+          export S3_IMAGE_TEST_DATA_PATH=$_WOLF_NPRD_GENERAL_S3_IMAGE_TEST_DATA_PATH
+          export JCLOUD_API=https://api-ci-episr.wolf.jina.ai
+          export DOCKER_PASSWORD=$_WOLF_NPRD_GENERAL_DOCKER_PASSWORD
+          export HUBBLE_AUTH_TOKEN=$_WOLF_NPRD_GENERAL_HUBBLE_AUTH_TOKEN
+          export DOCKER_USERNAME=$_WOLF_NPRD_GENERAL_DOCKER_USERNAME
+          export JINA_DEV_BOT=$_WOLF_NPRD_GENERAL_JINA_DEV_BOT
+          export JINA_OPTOUT_TELEMETRY=$_WOLF_NPRD_GENERAL_JINA_OPTOUT_TELEMETRY
+          export NETLIFY_AUTH_TOKEN1=$_WOLF_NPRD_GENERAL_NETLIFY_AUTH_TOKEN1
+          export NETLIFY_SITE_ID=$_WOLF_NPRD_GENERAL_NETLIFY_SITE_ID
+          export NOW_ANNLITE_EXECUTOR_SECRET=$_WOLF_NPRD_GENERAL_NOW_ANNLITE_EXECUTOR_SECRET
+          export NOW_AUTH_EXECUTOR_SECRET=$_WOLF_NPRD_GENERAL_NOW_AUTH_EXECUTOR_SECRET
+          export NOW_AUTOCOMPLETE_SECRET=$_WOLF_NPRD_GENERAL_NOW_AUTOCOMPLETE_SECRET
+          export NOW_ELASTIC_EXECUTOR_SECRET=$_WOLF_NPRD_GENERAL_NOW_ELASTIC_EXECUTOR_SECRET
+          export NOW_OCR_EXECUTOR_SECRET=$_WOLF_NPRD_GENERAL_NOW_OCR_EXECUTOR_SECRET
+          export NOW_POSTPROCESSOR_EXECUTOR_SECRET=$_WOLF_NPRD_GENERAL_NOW_POSTPROCESSOR_EXECUTOR_SECRET
+          export NOW_PREPROCESSOR_JCLOUD_TOKEN=$_WOLF_NPRD_GENERAL_NOW_PREPROCESSOR_JCLOUD_TOKEN
+          export NOW_PREPROCESSOR_REPO=$_WOLF_NPRD_GENERAL_NOW_PREPROCESSOR_REPO
+          export NOW_PREPROCESSOR_REPO_TOKEN=$_WOLF_NPRD_GENERAL_NOW_PREPROCESSOR_REPO_TOKEN
+          export NOW_QDRANT_EXECUTOR_SECRET=$_WOLF_NPRD_GENERAL_NOW_QDRANT_EXECUTOR_SECRET
+          export NOW_STAGING_FLORIAN=$_WOLF_NPRD_GENERAL_NOW_STAGING_FLORIAN
+          export PERSONAL_ACCESS_TOKEN=$_WOLF_NPRD_GENERAL_PERSONAL_ACCESS_TOKEN
+          export TWINE_PASSWORD=$_WOLF_NPRD_GENERAL_TWINE_PASSWORD
+          export TWINE_USERNAME=$_WOLF_NPRD_GENERAL_TWINE_USERNAME
+          export WOLF_EXAMPLES_TOKEN=$_WOLF_NPRD_GENERAL_WOLF_EXAMPLES_TOKEN
+          export WOLF_TOKEN=$_WOLF_NPRD_GENERAL_WOLF_TOKEN
+          export AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }}
+          export AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
Use `env` instead of using `export`s.

Reply:
I had some trouble with this when using `env`. Will the following syntax work here?
```
S3_IMAGE_TEST_DATA_PATH: $_WOLF_NPRD_GENERAL_S3_IMAGE_TEST_DATA_PATH
```
When secrets are pulled from the AWS Secret Manager the name under which the secret is stored is appended to the secret key

Reply:
Using `env` causes the tests to hang https://github.com/jina-ai/cloud-ops/actions/runs/3675012298/jobs/6213918956